schema_files/alterTables.sql

```diff
@@ -0,0 +1,88 @@
+--make changes to db for future features...
```

Comment:
So, this file should at least be numbered like "update_001" or something... and maybe the TQB_Dump.sql should be like "base" or "update_000" or something so it's clear what order to run things in? Assuming that is a dump *before* these updates...

Reply:
That's a good call.  The TQB_Dump.sql reflects the same schema as this, but is just the auto generated sql dump in case I missed something in this 'hand-written' alterTables.sql file.  I was considering having Mtil dump his db so that I could test that these changes in alterTables.sql work, but that juice might not be worth the squeeze.  Agree we should use sequelize or something in the future.